dds_web/api/user.py

```diff
@@ -218,11 +218,34 @@ def invite_user(args):
             unit_email=unit_email,
         )
 
-        mail.send(msg)
+        AddUser.send_email_with_retry(msg)
```

Comment:
Not `self`?

Reply:
Its a static function, so can't be called with `self`.

Reply:
Riiight nvm